test/L2ArbitrumGovernorV2.t.sol

```diff
@@ -196,6 +206,96 @@ abstract contract Quorum is L2ArbitrumGovernorV2Test {
   }
 }
 
+abstract contract Propose is L2ArbitrumGovernorV2Test {
+  event ProposalCreated(
+    uint256 proposalId,
+    address proposer,
+    address[] targets,
+    uint256[] values,
+    string[] signatures,
+    bytes[] calldatas,
+    uint256 voteStart,
+    uint256 voteEnd,
+    string description
+  );
+
+  function testFuzz_Propose(uint256 _actorSeed) public {
+    // Proposal parameters
+    address[] memory targets = new address[](1);
+    uint256[] memory values = new uint256[](1);
+    bytes[] memory calldatas = new bytes[](1);
+    string[] memory signatures = new string[](1);
+    uint256 voteStart = vm.getBlockNumber() + governor.votingDelay();
+    uint256 voteEnd = voteStart + governor.votingPeriod();
+    string memory description = "Test";
+
+    uint256 proposalId = governor.hashProposal(targets, values, calldatas, keccak256(bytes(description)));
+    address _actor = _getMajorDelegate(_actorSeed);
+    vm.prank(_actor);
+    vm.expectEmit();
+    emit ProposalCreated(proposalId, _actor, targets, values, signatures, calldatas, voteStart, voteEnd, description);
+    governor.propose(targets, values, calldatas, description);
+
+    assertEq(uint256(governor.state(proposalId)), uint256(IGovernor.ProposalState.Pending));
+  }
+}
```

Comment:
shall we add other proposal tests? e.g. RevertIf_DoesNotMeetProposalThreshold? I think this would be a different PR if we chose to do it.

Reply:
Yes! Will do it in another PR.